src/Components/Dropdown/Dropdown.tsx

```diff
@@ -1,111 +1,130 @@
-import {Autocomplete, Box, FormControl, FormHelperText, TextField} from '@mui/material';
-import InputLabel from 'Components/InputLabel';
+import CheckBoxIcon from '@mui/icons-material/CheckBox';
+import CheckBoxOutlineBlankIcon from '@mui/icons-material/CheckBoxOutlineBlank';
+import Autocomplete from '@mui/material/Autocomplete';
+import Box from '@mui/material/Box';
+import Checkbox from '@mui/material/Checkbox';
+import FormControl from '@mui/material/FormControl';
+import FormHelperText from '@mui/material/FormHelperText';
+import TextField from '@mui/material/TextField';
 import React from 'react';
 
-interface Props {
+export interface DropdownProps {
   id: string;
   label?: string;
   disabled?: boolean;
-  isTouched?: boolean;
   enableAutoComplete?: boolean;
-  returnValue?: boolean;
   multiple?: boolean;
   helperText?: string;
   errorMessage?: any;
   options: Array<{value: any; label: string}>;
-  value?: any;
   onChange?: any;
+  ListboxComponent?: any;
+  ListboxProps?: any;
```

Comment:
some of these variables are not used anywhere,Please remove the unused ones and also extend the interface from mui types